index.html

```diff
@@ -124,7 +124,7 @@ <h4 class="secs">34</h4>
 
     <!-- main program section -->
 
- <section class="program-section">
+ <section id="program-sec" class="program-section">
     <div class="program-sec-wrapper">
         <div class="program_heading">
```

Comment:
- [ ] Let's change the color text of this see program button or link 👇🏻 

![colortext52](https://user-images.githubusercontent.com/79058364/170736544-4e1c24db-1f75-461b-bd21-1e26ab17a8b5.png)